compensations/pom.xml

```diff
@@ -239,7 +237,22 @@
              </dependency>
 	  </dependencies>
       </profile>
-
+        <profile>
+            <id>jdk17</id>
+            <activation>
+                <jdk>[17,)</jdk>
+            </activation>
+            <properties>
+                <jvm.args.modular>
+                    --add-modules=java.se
```

Comment:
I am not sure if this java.se module required to be added here because I guess https://github.com/jbosstm/narayana/blob/master/compensations/pom.xml#L216 might already take effect?

Reply:
I will remove it from JDK17 profile